src/main/kotlin/no/nav/familie/tilbake/dokumentbestilling/varsel/VarselbrevUtil.kt

```diff
@@ -2,24 +2,34 @@ package no.nav.familie.tilbake.dokumentbestilling.varsel
 
 import no.nav.familie.kontrakter.felles.tilbakekreving.FeilutbetaltePerioderDto
 import no.nav.familie.kontrakter.felles.tilbakekreving.ForhåndsvisVarselbrevRequest
+import no.nav.familie.kontrakter.felles.tilbakekreving.Ytelsestype
 import no.nav.familie.tilbake.api.dto.FaktaFeilutbetalingDto
 import no.nav.familie.tilbake.behandling.domain.Behandling
 import no.nav.familie.tilbake.behandling.domain.Fagsak
 import no.nav.familie.tilbake.behandling.domain.Varsel
+import no.nav.familie.tilbake.beregning.TilbakekrevingsberegningService
 import no.nav.familie.tilbake.common.ContextService
+import no.nav.familie.tilbake.common.exceptionhandler.Feil
 import no.nav.familie.tilbake.config.Constants
 import no.nav.familie.tilbake.dokumentbestilling.felles.Adresseinfo
 import no.nav.familie.tilbake.dokumentbestilling.felles.Brevmetadata
 import no.nav.familie.tilbake.dokumentbestilling.felles.BrevmottagerUtil
 import no.nav.familie.tilbake.dokumentbestilling.felles.EksterneDataForBrevService
 import no.nav.familie.tilbake.dokumentbestilling.handlebars.dto.Handlebarsperiode
+import no.nav.familie.tilbake.dokumentbestilling.varsel.handlebars.dto.FeilutbetaltPeriode
 import no.nav.familie.tilbake.dokumentbestilling.varsel.handlebars.dto.Varselbrevsdokument
+import no.nav.familie.tilbake.dokumentbestilling.varsel.handlebars.dto.Vedleggsdata
 import no.nav.familie.tilbake.integration.pdl.internal.Personinfo
+import no.nav.familie.tilbake.integration.økonomi.OppdragClient
 import org.springframework.stereotype.Service
 import java.time.LocalDate
+import java.time.YearMonth
+import java.util.UUID
 
 @Service
-class VarselbrevUtil(private val eksterneDataForBrevService: EksterneDataForBrevService) {
+class VarselbrevUtil(private val eksterneDataForBrevService: EksterneDataForBrevService,
```

Comment:
Er det en util klass eller en service class?

Reply:
Det er en støtteklasse for VarselbrevService, så da tenkte jeg Util var et fint navn.